tests/at_functional_test/test/at_client_sync_test.dart

```diff
@@ -80,6 +80,7 @@ void main() {
     expect(updateResponse.isNotEmpty, true);
     // waiting for 15 seconds for sync to complete.
     await Future.delayed(Duration(seconds: 10));
+    atClientManager.atClient.syncService.sync();
```

Comment:
Seems like this call to sync() should be done **before** doing the wait for 10 seconds

Reply:
Updated